framework/src/consumption/lib/redshift-serverless/redshift-serverless-workgroup.ts

```diff
@@ -0,0 +1,268 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+import { createHash } from 'crypto';
+import { Names, RemovalPolicy, Stack } from 'aws-cdk-lib';
+import { Connections, IConnectable, Port, SecurityGroup, SelectedSubnets, SubnetSelection, Vpc } from 'aws-cdk-lib/aws-ec2';
+import { CfnConnection } from 'aws-cdk-lib/aws-glue';
+import { CfnWorkgroup } from 'aws-cdk-lib/aws-redshiftserverless';
+import { Construct } from 'constructs';
+import { RedshiftServerlessNamespace } from './redshift-serverless-namespace';
+import { RedshiftServerlessWorkgroupProps } from './redshift-serverless-workgroup-props';
+import { DataCatalogDatabase } from '../../../governance';
+import { Context, TrackedConstruct, TrackedConstructProps, Utils } from '../../../utils';
+import { RedshiftData } from '../redshift/redshift-data';
+
+const DEFAULT_PORT = 5439;
+
+/**
+ * Create a Redshift Serverless Workgroup. A default namespace would be created if none is provided.
+ *
+ * @example
+ * import { IpAddresses, SubnetType, Vpc } from 'aws-cdk-lib/aws-ec2';
+ * const vpc = new Vpc(this, "rs-example-network", {
```

Comment:
can you follow the new way of building documentation/examples we have the instructions on the [cookbook](https://awslabs.github.io/data-solutions-framework-on-aws/docs/constructs/cookbook/compiled-documentation)

Reply:
Added the examples